lib/routes/components.js

```diff
@@ -130,9 +153,9 @@ route = _.bindAll({
     switch (req.params.ext.toLowerCase()) {
       case 'html':
         return this.render(req, res);
-      case 'json': // jshint ignore:line
+      case 'json':
       default:
```

Comment:
technically these defaults are never hit, since we [check against a list of approved extensions](https://github.com/nymag/amphora/blob/master/lib/responses.js#L286) before this code is hit